### PR TITLE
Development

### PR DESCRIPTION
**Pull request by ghoshbishakh**

### CODE REVIEW

File: website/views/tools.py [hunk 1]
```diff
@@ -197,13 +198,58 @@ def update_documentations():
...
+                       url="/", image=settings.DEFAULT_LOGO_URL,
+                       object_type="website"):
+    """
+    Get meta data dictionary for a page
+
+    Parameters
+    ----------
+    title : string
+        The title of the page used in og:title, twitter:title, <title> tag etc.
+    description: string
```
Comment (Garyfallidis) [inline website/views/tools.py]: add an space between description and ``:``

File: website/views/tools.py [hunk 2]
```diff
@@ -197,13 +198,58 @@ def update_documentations():
...
+        Description used in description meta tag as well as the
+        og:description and twitter:description property.
+    keywords: list
```
Comment (Garyfallidis) [inline website/views/tools.py]: same here

File: website/views/tools.py [hunk 3]
```diff
@@ -197,13 +198,58 @@ def update_documentations():
...
+        List of keywords related to the page
+    url: string
```
Comment (Garyfallidis) [inline website/views/tools.py]: same

File: website/views/tools.py [hunk 4]
```diff
@@ -197,13 +198,58 @@ def update_documentations():
...
+        Full or partial url of the page
+    image: string
```
Comment (Garyfallidis) [inline website/views/tools.py]: same

File: website/views/tools.py [hunk 5]
```diff
@@ -197,13 +198,58 @@ def update_documentations():
...
+        Full or partial url of an image
+    object_type: string
```
Comment (Garyfallidis) [inline website/views/tools.py]: same

**Comment by Garyfallidis** [comment]
Hi @ghoshbishakh just some minor comments here. Please make the changes and will merge asap.

**Comment by ghoshbishakh** [comment]
@Garyfallidis I have made the changes.

**Comment by Garyfallidis** [comment]
Thank you :)

**Pull request merged**